ambry-tools/src/main/java/com.github.ambry/store/DumpData.java

```diff
@@ -138,6 +140,13 @@ public static void main(String args[]) {
               .defaultsTo("false")
               .ofType(String.class);
 
+      ArgumentAcceptingOptionSpec<Integer> bytesPerSecOpt =
+          parser.accepts("bytesPerSec", "Allowed bytes per sec for the purpose of throttling")
```

Comment:
I am planning to unify all these params in a config file which will be synonymous to how our frontend works, i.e. a separate config file. Will put up a new PR in few days